src/services/findAllReferences.ts

```diff
@@ -129,9 +129,7 @@ namespace ts.FindAllReferences {
         })();
 
         const { node, name, kind, displayParts } = info;
-        const sourceFile = node.getSourceFile();
-        const textSpan = getTextSpan(isComputedPropertyName(node) ? node.expression : node, sourceFile);
-        return { containerKind: ScriptElementKind.unknown, containerName: "", fileName: sourceFile.fileName, kind, name, textSpan, displayParts };
+        return { containerKind: ScriptElementKind.unknown, containerName: "", kind, name, displayParts, ...getTextSpanAndFileName(isComputedPropertyName(node) ? node.expression : node) };

```

Comment:
Using this helper means we are creating unnecessary object and throwing it rightaway. Given that this is called in so many places, i would rather have it inlined than create this indirection.